### ISSUE TITLE
Expose `ouput` directory location to transforms

### ISSUE THREAD

**Issue by sebastianbenz** [11ty/eleventy#922]
I'm trying to build a plugin that automatically inlines all CSS. In order to be able to resolve root-relative paths to local CSS files (e.g. `/css/main.css`) my transform needs to know the `output` directory's location.

Suggestion: expose the `output` directory via the `eleventyConfig` object passed to transform.

**Comment by keith24** [comment]
Isn't the output directory already exposed in `eleventyConfig`?  Seems so obvious that I probably don't understand what you're trying to do.  

```js
eleventyConfig.addTransform('my-transform', (content, outputPath) => {
  console.log(`Output dir: ${eleventyConfig.dir['output']}`)
  ...
})
```

That spits out my output dir.  I did set it explicitly with `eleventyConfig.dirs` though.

**Comment by sebastianbenz** [comment]
This unfortunately doesn't work for plugins. The output dir property is not set for the config passed to the plugin.

**Comment by zachleat** [comment]
(this is a note for me https://github.com/11ty/eleventy-plugin-vue/blob/master/.eleventy.js#L96 that you can do this inside of the new addExtension API but there are going to be some weird race conditions with this otherwise, since those dirs are returned from the config and addPlugin is called inside of the config)

**Comment by zachleat** [comment]
https://github.com/11ty/eleventy/pull/1320 fixed this issue and will ship with 1.0.

You can use `eleventyConfig.dir` in your plugins.

**Comment by Ryuno-Ki** [comment]
In case the referenced file changes over time, this is the content at time of publication:
https://github.com/11ty/eleventy-plugin-vue/blob/5cda73100a71c6a5df1ed66cf91ba9eb7ef695b4/.eleventy.js#L96
